### ISSUE TITLE
JSON Manager with eos model

### ISSUE THREAD

**Issue by kolibs** [kolibs/eos#2]
Need a json manager to create, update and delete content in a JSON file and get a line as a tuple.

**Comment by xavitator** [comment]
https://github.com/ocaml-community/yojson

**Comment by xavitator** [comment]
I think, for now, the best yojson module to use is [Yojson.Basic](https://mjambon.github.io/mjambon2016/yojson-doc/Yojson.Basic.html).

**Comment by kolibs** [comment]
I agree with you! I try Yojson during my tests with mustache :  simple and efficient.
